### PR TITLE
fmcjesdadc1: add ad9250 support

### PR DESCRIPTION
**Pull request by amiclaus**
1. Intergate AD9250 main driver into the fmcjesdadc1 project.
2. Remove ad9250 minimal implementation for the fmcjesdadc1 project.

Signed-off-by: Antoniu Miclaus <antoniu.miclaus@analog.com>

### CODE REVIEW

File: projects/fmcjesdadc1/src/app/fmcjesdadc1.c [hunk 1]
```diff
@@ -256,19 +329,27 @@ int main(void)
...
 	};
-	ad9250_test(ad9250_1_device, AD9250_TEST_PNLONG);
+	ad9250_test_mode(ad9250_1_device, 5);
+	ad9250_transfer(ad9250_1_device);
 	if(axi_adc_pn_mon(ad9250_1_core, AXI_ADC_PN23, 10) == -1) {
 		printf("%s ad9250_1 - PN23 sequence mismatch!\n", __func__);
 	};
 
 	// set up ramp output
-	ad9250_test(ad9250_0_device, AD9250_TEST_RAMP);
```
Comment (scuciurean) [inline projects/fmcjesdadc1/src/app/fmcjesdadc1.c:270]: You could keep the macros for the test modes.

Comment (amiclaus) [reply projects/fmcjesdadc1/src/app/fmcjesdadc1.c:270]: I agree. But the modification you suggest implies updating the ad9250 driver, while the PR handles the fmcjesdadc1 project update. I will create a separate commit/PR afterwards.

**Comment by amiclaus** [comment]
V2:
- fix Codacy complains

**Comment by amiclaus** [comment]
V3:
- add commits for test mode and output format macro definitions in the ad9250 driver
- use newly added macro definitions in the main functions of the fmcjesdadc1 project

**Pull request merged**